transport/transport.go

```diff
@@ -67,7 +79,7 @@ type Transport interface {
 
 // Listener is an interface closely resembling the net.Listener interface.  The
 // only real difference is that Accept() returns Conn's of the type in this
-// package, and also exposes a Multiaddr method as opposed to a regular Addr
+// package, and also exposes a Multiaddr method as well as a regular Addr
```

Comment:
I think the original doc was correct.

```suggestion
// package, and also exposes a Multiaddr method as opposed to a regular Addr
```